CODEOWNERS

```diff
@@ -34,75 +34,75 @@
+/doc/ref_cert/RC1/chapters/chapter06.md        @scottsteinbrueck @fuqiao123 @walterkozlowski
+/doc/ref_cert/RC1/chapters/chapter07.md        @scottsteinbrueck @fuqiao123 @walterkozlowski
+/doc/ref_cert/RC1/chapters/chapter08.md        @scottsteinbrueck @fuqiao123 @walterkozlowski
+/doc/ref_cert/RC1/chapters/chapter09.md        @scottsteinbrueck @fuqiao123 @walterkozlowski
+/doc/ref_cert/RC1/chapters/chapter10.md        @scottsteinbrueck @fuqiao123 @walterkozlowski
+/doc/ref_cert/RC1/chapters/chapter11.md        @scottsteinbrueck @fuqiao123 @walterkozlowski
 
 
 # Reference Architectures 2
-/doc/ref_arch/kubernetes/chapters/chapter01.md        @scottsteinbrueck @rgstori
-/doc/ref_arch/kubernetes/chapters/chapter02.md        @scottsteinbrueck @rgstori
-/doc/ref_arch/kubernetes/chapters/chapter03.md        @scottsteinbrueck @rgstori
-/doc/ref_arch/kubernetes/chapters/chapter04.md        @scottsteinbrueck @rgstori
-/doc/ref_arch/kubernetes/chapters/chapter05.md        @scottsteinbrueck @rgstori
-/doc/ref_arch/kubernetes/chapters/chapter06.md        @scottsteinbrueck @rgstori
-/doc/ref_arch/kubernetes/chapters/chapter07.md        @scottsteinbrueck @rgstori
-/doc/ref_arch/kubernetes/chapters/chapter08.md        @scottsteinbrueck @rgstori
+/doc/ref_arch/kubernetes/chapters/chapter01.md        @scottsteinbrueck @rgstori @walterkozlowski
+/doc/ref_arch/kubernetes/chapters/chapter02.md        @scottsteinbrueck @rgstori @walterkozlowski
+/doc/ref_arch/kubernetes/chapters/chapter03.md        @scottsteinbrueck @rgstori @walterkozlowski
+/doc/ref_arch/kubernetes/chapters/chapter04.md        @scottsteinbrueck @rgstori @walterkozlowski
+/doc/ref_arch/kubernetes/chapters/chapter05.md        @scottsteinbrueck @rgstori @walterkozlowski
+/doc/ref_arch/kubernetes/chapters/chapter06.md        @scottsteinbrueck @rgstori @walterkozlowski
+/doc/ref_arch/kubernetes/chapters/chapter07.md        @scottsteinbrueck @rgstori @walterkozlowski
+/doc/ref_arch/kubernetes/chapters/chapter08.md        @scottsteinbrueck @rgstori @walterkozlowski
 
 
 # Reference Implementation 2
-/doc/ref_impl/cntt-ri2/chapters/chapter01.md        @scottsteinbrueck @tomkivlin
-/doc/ref_impl/cntt-ri2/chapters/chapter02.md        @scottsteinbrueck @tomkivlin
-/doc/ref_impl/cntt-ri2/chapters/chapter03.md        @scottsteinbrueck @tomkivlin
-/doc/ref_impl/cntt-ri2/chapters/chapter05.md        @scottsteinbrueck @tomkivlin
-/doc/ref_impl/cntt-ri2/chapters/chapter04.md        @scottsteinbrueck @tomkivlin
-/doc/ref_impl/cntt-ri2/chapters/chapter06.md        @scottsteinbrueck @tomkivlin
-/doc/ref_impl/cntt-ri2/chapters/chapter07.md        @scottsteinbrueck @tomkivlin
-/doc/ref_impl/cntt-ri2/chapters/chapter08.md        @scottsteinbrueck @tomkivlin
-/doc/ref_impl/cntt-ri2/chapters/chapter09.md        @scottsteinbrueck @tomkivlin
+/doc/ref_impl/cntt-ri2/chapters/chapter01.md        @scottsteinbrueck @tomkivlin @walterkozlowski
```

Comment:
@walterkozlowski In this section tomkivlin should be replaced by @rihabbanday @jgu17

Reply:
@pgoyal01 I've already covered this in your previous comment: https://github.com/cntt-n/CNTT/pull/2311#discussion_r595993185